addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -0,0 +1,219 @@
+@use '../../variables/sizing';
+@use '../../utilities/type';
+@use '../../variables/typography';
+
+$l: sizing.rems(l); // 16
```

Comment:
@randallmorey @DhariniJeeva 

I am not quite understading what is the purpose of defining variables that are sizing. 
Also, I find a lack of consistency on the naming, i.e. `$l` for the size l, but `$size-xxs` for the size xxs.

Why not using directly? Unless I am missing something and taking as example line 21.

Why: `height: $l + $size-xxs`; rather than `height: sizing.rems(l) + sizing.rems(xxs);`?

Mentioning both of you because I think for the sake of consistency and mantainability we should choose a pattern and stick to it (unless specific cases).

Reply:
The helper is verbose so that's probably why the values get aliased.  As for naming, I agree this should be consistent.

Reply:
I see the aliased advantatge of cutting verbose. To avoid re-defining these in each file we need it, what do you guys think on defining these variables  within `_sizing.scss` file? so they can be use as `sizing.l`, `sizing.xss` or something like that?

Reply:
happy to add a ticket for this work and address it later. we use aliased format In another place as well.